Khan/wonder-blocks
# WB-377 Add spinner prop to Button

We need a spinner button for the discussions rewrite and since we're using this Button for other parts of the UI we'd like to keep things consistent.  This was part of the original spec, we just didn't have time to get to it during the initial implementing.

Test Plan:
- view the spinner button examples on https://deploy-preview-285--wonder-blocks.netlify.com/#!/Button
- edit one of the examples to see that secondary and tertiary kinds work correctly

I'm leaving support for buttons with `light={true}` as a TODO since we don't have designs for those yet.
